integration/src/test/java/edu/unc/lib/boxc/integration/factories/WorkFactory.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright 2008 The University of North Carolina at Chapel Hill
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package edu.unc.lib.boxc.integration.factories;
+
+import edu.unc.lib.boxc.model.api.objects.WorkObject;
+
+import java.util.Map;
+
+/**
+ * @author sharonluong
```

Comment:
since these are test utilities, it would be good to add a brief description of what these factories are for (something like "Factory for creating test WorkObjects and their files")